api/src/extensions/manager.ts

```diff
@@ -794,4 +792,20 @@ export class ExtensionManager {
 
 		await Promise.all(unregisterFunctions.map((fn) => fn()));
 	}
+
+	/**
+	 * If extensions must load successfully, any errors will cause the process to exit.
+	 * Otherwise, the error will only be logged as a warning.
+	 */
+	private handleExtensionError(type: ExtensionType, name: string, error: any): void {
+		if (toBoolean(env['EXTENSIONS_MUST_LOAD'])) {
+			logger.error('EXTENSION_MUST_LOAD is enabled and an extension failed to load.');
+			logger.error(`Couldn't register ${type} "${name}"`);
+			logger.error(error);
+			process.exit(0);
```

Comment:
Is there a specific reason to exit with `0` or should we
```suggestion
			process.exit(1);
```
?

Reply:
No specific reason, in hindsight `1` is more consistent with other similar "exit server" logics indeed!